subsys/bluetooth/controller/hci_internal.c

```diff
@@ -1358,6 +1360,7 @@ int hci_internal_cmd_put(uint8_t *cmd_in)
 		 */
 
 		cmd_complete_or_status.occurred = false;
+		padv_response_data_cmd_pending = true;
 	}
```

Comment:
may be we should put that entire `if (opcode == SDC_HCI_OPCODE_CMD_LE_SET_PERIODIC_ADV_RESPONSE_DATA` under corresponding KConfig ifdef. Currently it eats few cycles for every event we generate, even for an application that has nothing to do with PAwR.